### PR TITLE
editorial: Use Web IDL's definition conventions for methods and getters.

### PR DESCRIPTION
**Pull request by rakuco**
Aligns with whatwg/webidl#882:
- Describe getters as "The attr getter steps are [...]".
- Describe methods as "The myMethod(arg) method steps are [...]".

***
<a href="https://pr-preview.s3.amazonaws.com/rakuco/battery/pull/49.html" title="Last updated on Nov 5, 2021, 12:54 PM UTC (c4d9c73)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/battery/49/4291db6...rakuco:c4d9c73.html" title="Last updated on Nov 5, 2021, 12:54 PM UTC (c4d9c73)">Diff</a>

### CODE REVIEW

**Review by anssiko — APPROVED** [review]
Thanks for this clean up. A less convoluted and more readable way.

**Comment by rakuco** [comment]
@marcoscaceres @xfq deploying this change to W3C is [failing](https://github.com/w3c/battery/runs/4119078735):

```
  Specberus Errors:
    The &lt;details&gt; element in the header should have attribute "open".
      {
        name: 'headers.details-summary',
        section: 'front-matter',
        rule: 'docIDFormat'
      }
    No GitHub repository issue link. The link is expected to be of the form <code>https://github.com/&lt;USER_OR_ORG&gt;/&lt;REPO_NAME&gt;/issues[/&hellip;]</code>.
      {
        name: 'headers.github-repo',
        section: 'front-matter',
        rule: 'gitRepo'
      }
  
  Details: https://labs.w3.org/echidna/api/status?id=ba689a50-51a8-4350-9a88-6d9520adcd1c
```

This check in specberus seems to be related to Process 2021, and I see some recent related ReSpec changes. Any ideas?

**Comment by xfq** [comment]
The reason for the first error is that the `details` element does not contain the `open` attribute (it seems to be added to the DOM by https://www.w3.org/scripts/TR/2021/fixup.js , but it is not in the static HTML file).

The reason for the second error is that the GitHub repo link should appear under `Feedback:` instead of `Participate`.

Will report ReSpec issues.

**Comment by rakuco** [comment]
Thanks, @xfq. FTR, I've fixed the second error with #50.

**Comment by anssiko** [comment]
@rakuco please merge when all errors have been fixed.

**Comment by rakuco** [comment]
@anssiko are you referring to this PR or do you mean I should've waited for the `details.open` bug to be fixed before merging #50?

**Comment by anssiko** [comment]
All is good now 👍 Feel free to merge fixes needed to keep the build clean without blocking on my review. Sometimes the build breaks and we need to deploy fixes fast so a review from whoever is available is good.

**Comment by xfq** [comment]
FTR, the first error was also fixed.

**Review by marcoscaceres — COMMENTED** [review]
Little late... but ✅

**Pull request merged**
